docs/source/usage/index.rst

```diff
@@ -88,6 +88,48 @@ Users can view available offers and contract a node from an offer.
 | Delete Contract | ``openstack lease contract delete <contract_uuid>``                                                      |
 +-----------------+----------------------------------------------------------------------------------------------------------+
 
+Provisioning a Node
+-------------------
+
+There are multiple ways for a non-admin to provision a node.
+
+Image
+~~~~~
+
+Image-based provisioning can be accomplished through the use of `Metalsmith`_. It requires the image to be uploaded into OpenStack Glance. Once that's done, a non-admin can run the following:
+
++----------------+---------------------------------------------------------------------------------------------------------------------+
+|                | **Actions**                                                                                                         |
++----------------+---------------------------------------------------------------------------------------------------------------------+
+| Provision Node | ``metalsmith deploy --resource-class baremetal --image <image> --network <network> --ssh-public-key <path-to-key>`` |
++----------------+---------------------------------------------------------------------------------------------------------------------+
+
+Volume
+~~~~~~
+
+The process for booting a node from a volume is described in the `Ironic boot-from-volume documentation`_. There are two things to note for non-admin deployments:
+
+* The node owner or admin should set the `iscsi_boot` node capability prior to leasing the node.
```

Comment:
Maybe an example command will help understand where to set iscsi_boot? (I know little about boot from volume)

Reply:
Fair point! I left this out deliberately because I want people to just go to the Ironic boot-from-volume doc link (where this is explained); otherwise we end up duplicating a ton of documentation.

Reply:
Got it. It makes sense to me now.